src/nplinker/genomics/antismash/antismash_loader.py

```diff
@@ -97,7 +98,7 @@ def get_bgcs(self) -> list[BGC]:
         return self._bgcs
 
     @staticmethod
-    def _parse_bgcs(bgc_files: dict[str, str]) -> list[BGC]:
+    def _parse_bgcs(bgc_files: Mapping[str, str]) -> list[BGC]:
```

Comment:
Why here not using `Sequence[BGC]` for the returned type?

Reply:
The principle is keeping robust for input (arguments) and keeping specific for output.
That is why it's nice to use wider type for input values but specific type for returned values.